atsign-foundation/atsign.dev-3.0
# feat: add iot staging build to deploy staging

**- What I did**

- Added deployment script for staging/iot

Planned deployments:
| REPO | CNAME |
| - | - |
| atsign.dev | dev3.atsign.wtf |
| iot.atsign.dev | iot3.atsign.wtf |
| atsign.dev-prod | atsign.dev |
| iot.atsign.dev-prod | iot.atsign.dev |

**- How I did it**

Local changes

**- How to verify it**

Need to setup the `iot.atsign.dev` repo and merge to ensure that the script works as expected.

**- Description for the changelog**
feat: add iot staging build to deploy staging
